cmd/csi-attacher/main.go

```diff
@@ -198,13 +198,31 @@ func main() {
 			klog.Error(err.Error())
 			os.Exit(1)
 		}
+
+		supportsSingleNodeMultiWriter, err := supportsSingleNodeMultiWriter(ctx, csiConn)
```

Comment:
can you please add it into `supportsControllerPublish()` above (and maybe rename it), so the attacher calls `GetControllerCapabilities` only once?

Reply:
I consolidated this logic into one call and put it in a separate commit, PTAL